### DISCUSSION TITLE
Precise orbit files

### DISCUSSION THREAD

**Discussion by Abdallahbalushi** [gmtsar/gmtsar#851] [Q&A]
When trying to download precise orbit files using organize_files_tops.csh command, it keeps skipping the data as in the images below. Any idea on the issue ??
![image](https://github.com/gmtsar/gmtsar/assets/150544398/f64fcb07-7387-4680-9d22-b7ff02db2ce5)
![image](https://github.com/gmtsar/gmtsar/assets/150544398/08517686-436d-4e67-8c2a-d22883a6d50a)
![image](https://github.com/gmtsar/gmtsar/assets/150544398/44799891-58ba-435a-b41d-043533e655cb)

**Comment by Xiaohua-Eric-Xu** [comment]
The issue is caused by ESA stopped their service on Sentinel data hub and moved everything to a new server named dataspace.copernicus.eu. So far I have not yet found how to access the orbit files without registering on the web service and querying on-line. You may want to switch to an earlier version of the script to download things automatically from ASF instead for the moment. e.g., https://github.com/gmtsar/gmtsar/blob/6.1/gmtsar/csh/organize_files_tops.csh.
